### PR TITLE
TGBOT-42-create-object-provider-for-car-objects

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-42
Create CarProvider and CarProviderTest

### CODE REVIEW

File: stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java [hunk 1]
```diff
@@ -25,6 +25,7 @@ public Car(String id, String brand, String model, String generation, String modi
         this.year = year;
     }
 
+
```
Comment (max-java) [inline stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java:28]: delete empty row

File: stoservice/src/test/java/com/tutrit/stoservice/provider/CarProviderTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.tutrit.stoservice.provider;
```
Comment (max-java) [inline stoservice/src/test/java/com/tutrit/stoservice/provider/CarProviderTest.java:1]: Remove tests

**Review by max-java — DISMISSED** [review]
remove redundant test, otherwise looks good

**Review by yurasava — APPROVED** [review]

**Review by lamaconda — APPROVED** [review]
lgfm

File: stoservice/src/main/java/com/tutrit/stoservice/provider/CarProvider.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarProvider {
+    public List<Car> getCar() {
+        List<Car> cars = new ArrayList<>();
+        for (int i = 1; i <= 5; i++) {
+            Car init = getInstance();
+            init.setId("id" +i);
```
Comment (schneidervs) [inline stoservice/src/main/java/com/tutrit/stoservice/provider/CarProvider.java]: please use the keyboard shortcut Ctrl+Shift+Alt+L in IDEA

**Comment by schneidervs** [comment]
LGFM

**Review by schneidervs — APPROVED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]
wonderful

**Review by yurasava — APPROVED** [review]
lgfm

**Pull request merged**